ray-operator/controllers/ray/utils/httpproxy_httpclient.go

```diff
@@ -57,10 +57,13 @@ func (r *RayHttpProxyClient) CheckProxyActorHealth(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		io.Copy(io.Discard, resp.Body)
```

Comment:
Read to EOF before closing it according to [client.Do](https://cs.opensource.google/go/go/+/refs/tags/go1.24.2:src/net/http/client.go;l=586).
```
// If the returned error is nil, the [Response] will contain a non-nil
// Body which the user is expected to close. If the Body is not both
// read to EOF and closed, the [Client]'s underlying [RoundTripper]
// (typically [Transport]) may not be able to re-use a persistent TCP
// connection to the server for a subsequent "keep-alive" request.
```